docs/quickstart.md

```diff
@@ -135,46 +173,76 @@ def cross_entropy(out, Y):
   y = y.reshape(list(Y.shape)+[num_classes])
   y = Tensor(y)
   return out.mul(y).mean()
+
```

Comment:
Depending on when/if #1018 is merged this will have to change to match.

Reply:
yes